mm2src/crypto/src/global_hd_ctx.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(non_local_definitions)]
```

Comment:
Can we add a todo or an issue about this too?

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/issues/2521